pages/edit.js

```diff
@@ -0,0 +1,30 @@
+import styled from "styled-components";
+import Link from "next/link";
+import UpdateFlashcard from "@/components/UpdateFlashcard";
+import { useRouter } from "next/router";
+
+const Container = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+`;
+
+export default function Edit({ handleUpdateFlashcard, flashcards }) {
+  const router = useRouter();
+  const { id } = router.query;
+  const flashcard = flashcards.find(function (flashcard) {
+    return flashcard.id === id;
+  });
```

Comment:
you can use the function keyword to define the callback function, but it might be better to stay consistent with rest of the codebase, where you use arrow functions.
```suggestion
  const flashcard = flashcards.find((flashcard) => flashcard.id === id);
```

Reply:
Done